oxidation/crates/parsec_api_protocol/schema/authenticated_cmds/user_get.json

```diff
@@ -0,0 +1,63 @@
+{
+    "label": "UserGet",
+    "req": {
+        "cmd": "user_get",
+        "other_fields": [
+            {
+                "name": "user_id",
+                "type": "UserID"
+            }
+        ]
+    },
+    "reps": [
+        {
+            "status": "ok",
+            "other_fields": [
+                {
+                    "name": "user_certificate",
+                    "type": "Bytes"
+                },
+                {
+                    "name": "revoked_user_certificate",
+                    "type": "Bytes"
+                },
+                {
+                    "name": "device_certificates",
+                    "type": "List<Bytes>"
+                },
+                {
+                    "name": "trustchain",
+                    "type": "Trustchain"
+                }
+            ]
+        },
+        {
+            "status": "not_found",
+            "other_fields": []
+        }
+    ],
+    "nested_types": [
+        {
+            "label": "Trustchain",
+            "variants": [
```

Comment:
nested type is required to be a variant ? having this `name: ""` field is confusing
it would be better to have optional `variants` and `schema` fields (and sanity check to refuse using both at the same time)

Reply:
also `tag` should be required when using `variants` (we want to be explicit !) and forbidden for `schema` (sanity check)